data/webassembly.json

```diff
@@ -0,0 +1,15 @@
+{
+  "editors": "https://webassembly.github.io/spec/",
+  "title": "WebAssembly",
+  "wgs": [
+    {
+      "label": "WebAssembly Working Group",
+      "url": "http://webassembly.org/"
```

Comment:
Although that page is much more useful that the rather empty and dull home page of the WG, I think we should still stick to the official home page of the WG: https://www.w3.org/wasm/

Hopefully, that page will soon link to http://webassembly.org!